src/sinks/mod.rs

```diff
@@ -133,200 +133,249 @@ pub enum HealthcheckError {
 #[serde(tag = "type", rename_all = "snake_case")]
 #[enum_dispatch(SinkConfig)]
 pub enum Sinks {
-    /// AMQP.
+    /// Send events to AMQP 0.9.1 compatible brokers like RabbitMQ.
     #[cfg(feature = "sinks-amqp")]
+    #[configurable(metadata(docs::label = "AMQP"))]
     Amqp(amqp::AmqpSinkConfig),
 
-    /// Apex Logs.
+    /// Deliver log events to Apex.
     #[cfg(feature = "sinks-apex")]
+    #[configurable(metadata(docs::label = "Apex"))]
     Apex(apex::ApexSinkConfig),
 
-    /// AWS CloudWatch Logs.
+    /// Publish log events to AWS CloudWatch Logs.
     #[cfg(feature = "sinks-aws_cloudwatch_logs")]
+    #[configurable(metadata(docs::label = "AWS CloudWatch Logs"))]
     AwsCloudwatchLogs(aws_cloudwatch_logs::CloudwatchLogsSinkConfig),
 
-    /// AWS CloudWatch Metrics.
+    /// Publish metric events to AWS CloudWatch Metrics.
     #[cfg(feature = "sinks-aws_cloudwatch_metrics")]
+    #[configurable(metadata(docs::label = "AWS CloudWatch Metrics"))]
     AwsCloudwatchMetrics(aws_cloudwatch_metrics::CloudWatchMetricsSinkConfig),
 
-    /// AWS Kinesis Firehose.
+    /// Publish logs to AWS Kinesis Data Firehose topics.
     #[cfg(feature = "sinks-aws_kinesis_firehose")]
+    #[configurable(metadata(docs::label = "AWS Kinesis Data Firehose Logs"))]
     AwsKinesisFirehose(aws_kinesis::firehose::KinesisFirehoseSinkConfig),
 
-    /// AWS Kinesis Streams.
+    /// Publish logs to AWS Kinesis Streams topics.
     #[cfg(feature = "sinks-aws_kinesis_streams")]
+    #[configurable(metadata(docs::label = "AWS Kinesis Streams Logs"))]
     AwsKinesisStreams(aws_kinesis::streams::KinesisStreamsSinkConfig),
 
-    /// AWS S3.
+    /// Store observability events in the AWS S3 object storage system.
     #[cfg(feature = "sinks-aws_s3")]
+    #[configurable(metadata(docs::label = "AWS S3"))]
     AwsS3(aws_s3::S3SinkConfig),
 
-    /// AWS SQS.
+    /// Publish observability events to Simple Queue Service topics.
```

Comment:
```suggestion
    /// Publish observability events to AWS Simple Queue Service topics.
```

**suggestion (non-blocking)**: You kept the AWS name in the other descriptions, keep it here for consistency?